api/src/info.rs

```diff
@@ -141,14 +141,14 @@ impl MemoryRegion {
 /// Represents the different types of memory.
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
 #[non_exhaustive]
-#[repr(C)]
+#[repr(C, u64)]
 pub enum MemoryRegionKind {
     /// Unused conventional memory, can be used by the kernel.
     Usable,
     /// Memory mappings created by the bootloader, including the page table and boot info mappings.
     ///
     /// This memory should _not_ be used by the kernel.
-    Bootloader,
+    Bootloader = 0x8FFFFFFF,
```

Comment:
It appeared to be conflicting with UEFI boots, since Bootloader and LOADER_CODE both = 1
But that wasn't the case, and may be my misunderstanding of rusts enums. I'll undo this.